examples/Azure_IoT_Central_ESP32/Azure_IoT_Central_ESP32.ino

```diff
@@ -407,13 +412,22 @@ void loop()
   if (WiFi.status() != WL_CONNECTED)
   {
     azure_iot_stop(&azure_iot);
+    
     connect_to_wifi();
+    
+    if (!azure_initial_connect)
+    {
+      configure_azure_iot();
+    }
+    
+    azure_iot_start(&azure_iot);
   }
   else
   {
     switch (azure_iot_get_status(&azure_iot))
     {
       case azure_iot_connected:
+        if (!azure_initial_connect) azure_initial_connect = true;
```

Comment:
why check before setting? Is this just adding more code for the check?

Reply:
Fair. Fixed.